Makefile

```diff
@@ -187,7 +187,12 @@ publish_mix: compile
 
 #==> Tests tasks
 
-test: test_erlang test_elixir
+test:
+	$(RM) -r .home
+	mkdir .home
+	HOME=$(CURDIR)/.home $(MAKE) test_erlang
```

Comment:
or, I can also just set `$HOME` to something sensible direcly in the debian packaging; that is probably easier for everyone.